third_party/NordicSemiconductor/libraries/nrf_security/config/nrf-config.h

```diff
@@ -653,15 +743,15 @@
  *
  * Comment macros to disable the curve and functions for it
  */
-#define MBEDTLS_ECP_DP_SECP192R1_ENABLED
-#define MBEDTLS_ECP_DP_SECP224R1_ENABLED
+/* #undef MBEDTLS_ECP_DP_SECP192R1_ENABLED */
```

Comment:
Please take a look at https://github.com/openthread/openthread/pull/5265. Do you know if the additional curves are present in DTLS client hello? If still, we should apply changes as in https://github.com/openthread/openthread/pull/5265.

Reply:
Discussed with @piotrkoziar and this is something that can be addressed at a later time. If this happens to be still applicable it is only a small inconsistency in the libs provided and configuration file.